Chore: Clean up benchmarks website

connortsui20 opened the pull request:
Because lance is so much slower in some of the queries than every other target, it makes a lot of charts harder to see everything.

EDIT: They are actually not showing up at all on the TPCH

EDIT AGAGIN: I'm going to use this PR to fix the two other problems. The first is that because we do not run lance on every PR the summaries will disappear. The second is that TPC-H queries are not showing up.

connortsui20 committed:
hide lance from benchmarks by default

Signed-off-by: Connor Tsui <connor.tsui20@gmail.com>

connortsui20 committed:
fix scoring

Get latest benchmark results per target instead of per commit.

Signed-off-by: Connor Tsui <connor.tsui20@gmail.com>

connortsui20 committed:
modularize the benchmark website (better?)

Signed-off-by: Connor Tsui <connor.tsui20@gmail.com>

connortsui20 committed:
fix missing TPC-H data

Signed-off-by: Connor Tsui <connor.tsui20@gmail.com>

robert3005 commented:
I am surprised we don't see some negative lines here. We rewrote the handling but old code should be removed? I doubt we just added 3k lines here

connortsui20 committed:
Remove unused modular system

The modular system was added but never actually used in production.
It added 3,275 lines of dead code that only increased bundle size.

Keeping only the essential bug fixes:
- scoring.js: Fixed to handle missing benchmark data
- data-shared.js: Fixed case-sensitivity in renaming
- index.html: Lance hiding configuration

Removed:
- 14 unused modular system files (2,657 lines)
- Test and example files that shouldn't be in production (618 lines)

Signed-off-by: Connor Tsui <connor.tsui20@gmail.com>

connortsui20 commented:
Turns out it was just a lot of dead code
